project/src/helper/helper.js

```diff
@@ -0,0 +1,9 @@
+function constructId(title) {
+    return title.split(/[ |,|]/).join('');
+}
+
+function constructIdFieldset(legend, label) {
+    const newLegend = legend.split(/[ |,|]/).join('');
```

Comment:
Is there any already declared function you could use here?

Reply:
👍